contrib/scripts/kind.sh

```diff
@@ -9,7 +9,11 @@ default_workers=1
 default_cluster_name=""
 default_image=""
 default_kubeproxy_mode="iptables"
-default_ipfamily="dual"
+if [ $(sysctl -n net.ipv6.conf.all.disable_ipv6) == 1 ] ; then
```

Comment:
I believe this only works on Linux, which isn't great for macOS devs. If anything, I think we should keep the default as is, and possibly chose a smarter option if on Linux and the default is unchanged.

Reply:
addressed with https://github.com/cilium/cilium/pull/30332/commits/24dd1a5ac0796f48f7d2c6fb81c237036e75a8b0